requirements/base.txt

```diff
@@ -1,14 +1,22 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
Nice work splitting the PR into commits; made it really easy to review. The requirements upgrade especially is nice to have separate from the other changes.